src/quacc/atoms/skzcam.py

```diff
@@ -13,12 +13,489 @@
 from monty.io import zopen
 from monty.os.path import zpath
 
+
+class ElementInfo(TypedDict):
+    core: int
+    basis: str
+    ecp: str
+    ri_scf_basis: str
+    ri_cwft_basis: str
+
+
 if TYPE_CHECKING:
+    from ase.atom import Atom
     from numpy.typing import NDArray
 
 has_chemshell = find_spec("chemsh") is not None
 
 
+def create_orca_eint_blocks(
+    embedded_adsorbed_cluster: Atoms,
+    quantum_cluster_indices: list[int],
+    ecp_region_indices: list[int],
+    element_info: ElementInfo | None = None,
+    pal_nprocs_block: dict[str, int] | None = None,
+    method_block: dict[str, str] | None = None,
+    scf_block: dict[str, str] | None = None,
+    ecp_info: dict[str, str] | None = None,
+    include_cp: bool = True,
+    multiplicity: dict | None = None,
```

Comment:
It was also hard to tell what this was supposed to be. Looks like it should really be

```python
MultiplicityDict(TypedDict):
    ad_slab: int
    slab: int
    ad: int
```

and then `MultiplicityDict | None = None`.

I'd also recommend using more complete phrasing, e.g. `adsorbate_slab`, `slab`, `adsorbate`. The extra characters are worth it for clarity. Finally, the name `multiplicity` for the keyword argument is a bit misleading because one would naively think that's a single `int`, but it's not. I would recommend changing this to `multiplicities` at the very least.